Improve feed item richtext content

anita-chouhan opened the pull request:
Task: [https://app.asana.com/1/1158496955069020/project/1203079866666008/task/1209807427613170?focus=true](https://app.asana.com/1/1158496955069020/project/1203079866666008/task/1209807427613170?focus=true)

Note: The `typogrify` filter is replaced with `typogrifyFeed` as the previous one was wrapping the `&amp;` characters in a separate span, which is not needed.

 **Reference:**
- _https://github.com/miranj/miranj.in/pull/86_

anita-chouhan committed:
Improve feed item richtext content

anita-chouhan committed:
Merge branch 'dev' into bug/double-esacape-quotes

anita-chouhan committed:
Merge remote-tracking branch 'origin/release/3.5.0' into bug/double-esacape-quotes

rungta commented:
@anita-chouhan Why are there composer and project config changes this PR? Also the Changelog entry seems duplicated. Perhaps the base branch here is wrong?

rungta committed:
Merge branch 'dev' into bug/double-esacape-quotes

rungta committed:
Fix regression

- Avoid applying enkode_emails twice
- Avoid forcing markdownExtra on feed content

rungta committed:
Improve changelog
